### DISCUSSION TITLE
Can't clone repository, bad tree object

### DISCUSSION THREAD

**Discussion by darkshade9** [community/community#29942] [Repositories]
Hello,

I've been working in my repository (a fork of another) recently and have been running into some strange behavior when cloning and checking out one of my branches.  Today, I tried a fresh clone of the repository and was greeted with this:

```
remote: fatal: bad tree object 753bcfd436c9bbfa4ad9b5316b688868b201a6ef
remote: aborting due to possible repository corruption on the remote side.
fatal: early EOF
fatal: index-pack failed
```

How do I contact a member of the Github team to resolve this, it appears it is a problem on the remote end?  I found [this SO post](https://stackoverflow.com/questions/19234581/github-clone-error) about how this user resolved their issue by contacting support, but the UI here doesn't make it easy to find said support contact information.

I've performed `git fsck` and `git gc` locally and I couldn't push up to my branch, which prompted me to attempt a fresh clone:

```
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 16 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 497 bytes | 248.00 KiB/s, done.
Total 5 (delta 4), reused 1 (delta 0)
remote: Resolving deltas:  75% (3/4), completed with 3 local objects.
remote: fatal: pack has 1 unresolved delta
```

**Comment by airtower-luna** [comment]
Yeah, that looks like a bug on their side. The bug report form is here: https://support.github.com/contact/bug-report

**Comment by Timibreez** [comment]
You could try cloning the repository using SSH, that should do the trick. And please ensure you are not trying to clone into a git initialized environment. Good luck.

**Comment by rubenmoor** [comment]
same:

```
$ git pull
remote: fatal: bad tree object 3802a9789dc7bdaf583ebe909205faf9731cbf6a
remote: aborting due to possible repository corruption on the remote side.
fatal: protocol error: bad pack header
```

Now trying to reproduce with a fresh clone and it works again, including `git pull`. Weird bug.

**Comment by kadamsuraj1128** [comment]
fatal: bad tree object 5a1b0159ad7f7156a309b4c05a9cb5fa8c05009d
fatal: the remote end hung up unexpectedly
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
